### PR TITLE
Improve the installation section

### LINKED ISSUE #247: Add the open ssl section back to install guide
https://github.com/luckyframework/website/commit/4e2d88eac03023025f59f90ee631337ff1559021#r37885130

Turns out you do still need it :(

### PR DESCRIPTION
**Pull request by paulcsmith**
* Give each dependency its own section.
* Make code blocks clearer
* Clean up some wording
* Fixes some style issues

Also fixes #247

### CODE REVIEW

File: src/actions/guides/getting-started/installing.cr [hunk 1]
```diff
@@ -29,51 +27,127 @@ class Guides::GettingStarted::Installing < GuideAction
...
+    [Postgres CLI tools](https://postgresapp.com/documentation/cli-tools.html) are installed
+
+    ```plain
+    sudo mkdir -p /etc/paths.d &&
+      echo /Applications/Postgres.app/Contents/Versions/latest/bin | sudo tee /etc/paths.d/postgresapp
+    ```
+
+    There are other installation methods available in [Postgres CLI tools docs](https://postgresapp.com/documentation/cli-tools.html)
+
+    ## Linux dependencies
```
Comment (jwoertink) [inline src/actions/guides/getting-started/installing.cr]: Are these for Crystal? Or things Lucky needs if you're running on Linux?

File: src/actions/guides/getting-started/installing.cr [hunk 2]
```diff
@@ -29,51 +27,127 @@ class Guides::GettingStarted::Installing < GuideAction
...
+
+    Once the required dependencies above are installed, set up Lucky for your system.
+
+    ### Install Homebrew
+
+    Installation instructions from the [Homebrew website](https://brew.sh/)
+
+    ```plain
+    sudo mkdir -p /etc/paths.d &&
+    echo /Applications/Postgres.app/Contents/Versions/latest/bin | sudo tee /etc/paths.d/postgresapp
```
Comment (jwoertink) [inline src/actions/guides/getting-started/installing.cr]: These are the wrong instructions for this section

File: src/actions/guides/getting-started/installing.cr [hunk 3]
```diff
@@ -29,51 +27,127 @@ class Guides::GettingStarted::Installing < GuideAction
...
+    cd lucky_cli
+    ```
+
+    ### Check out the latest released version
+
+    ```plain
+    git checkout #{LuckyCliVersion.current_tag}
+    ```
+
+    > See a list of [released version](https://github.com/luckyframework/lucky_cli/releases)
```
Comment (jwoertink) [inline src/actions/guides/getting-started/installing.cr]: Not sure why, but reading down this, this part threw me off. I thought that was the end of the instructions, and I thought "what about building the binary?". Then I went back and looked and went "oh, there's more".. Maybe just make this text inside of a side note?

**Review by jwoertink — CHANGES_REQUESTED** [review]
Not sure why I didn't get notified about this before.. Or maybe I did and missed it? Overall this looks really nice. Only 1 major issue, but the rest looks decent.

File: src/actions/guides/getting-started/installing.cr (hunk 1 above)
Comment (paulcsmith) [reply src/actions/guides/getting-started/installing.cr]: Both I think...I don't know though..

I'm guessing these should be installed before anything else so I'll move it to the top

File: src/actions/guides/getting-started/installing.cr (hunk 2 above)
Comment (paulcsmith) [reply src/actions/guides/getting-started/installing.cr]: Oops :)

File: src/actions/guides/getting-started/installing.cr (hunk 3 above)
Comment (paulcsmith) [reply src/actions/guides/getting-started/installing.cr]: Yeah good idea! It's not that important so can be a regular text note

**Comment by paulcsmith** [comment]
After going through this I realized I should probably flip this around a bit. Install Crystal and the CLI, then install yarn and all that since those are optional deps anyway. Will ping once that's ready. 

THanks for the review!

**Comment by paulcsmith** [comment]
Ok I redid a bunch of stuff. LMK what you think

File: src/actions/guides/getting-started/installing.cr [hunk 4]
```diff
@@ -29,54 +163,63 @@ class Guides::GettingStarted::Installing < GuideAction
...
+    There are other installation methods available in [Postgres CLI tools docs](https://postgresapp.com/documentation/cli-tools.html)
+
+    ### 2. Ensure Postgres CLI tools installed
+
+    ```plain
+    psql --version
+    ```
 
-    ### Dependencies for browser tests
+    Should return `psql (PostgreSQL) 12.x` or higher
```
Comment (jwoertink) [inline src/actions/guides/getting-started/installing.cr]: Avram works on 9.6 or later, but maybe we just say 10.x or higher? 12 is really new, so some people may think "oh, I have to upgrade postgres to run this?"

**Review by jwoertink — APPROVED** [review]
This looks better 👍  I think it's good having that optional stuff at the bottom there. Plus at the top it makes it easier for people to skip to their setup right away.

File: src/actions/guides/getting-started/installing.cr (hunk 4 above)
Comment (paulcsmith) [reply src/actions/guides/getting-started/installing.cr]: Good point! I think 10.x sounds good

**Pull request merged**